donut/modules/courses/templates/scheduler.html

```diff
@@ -461,7 +486,58 @@ <h1>Course Scheduler</h1>
     function displayUnits() {
       units.text('Units: ' + String(coursesUnits))
     }
-
+  $('#modal').on('show.bs.modal', function(e) {
+    var courseNum = $(e.relatedTarget).data("courseNum");
+    var courseId = $(e.relatedTarget).data("courseId");
+    displayNotes(courseNum, courseId); 
+  });
+  $('#editButton').click(function(e) {
+    $('#postNotes').show();
+    $('#viewNotes').hide();
+    $('#editButton').hide();
+  });
+  $('#postNotes').submit(function(e) {
+    e.preventDefault();
+    $.ajax({
+      url: '/1/scheduler/edit_notes',
+      type: 'POST',
+      data:$('#postNotes').serialize(),
+      success:function(){
+        displayNotes($('#courseNum').val(), $('#courseId').val());
```

Comment:
If this returns an error message, you should display it

Reply:
Like this? https://github.com/ASCIT/donut/blob/1319a70e3f221b70e57235200c22d003a183f13c/donut/modules/uploads/static/script_uploading.js#L28

Reply:
No, it will still be in the `success` callback, but just check `data.success` and display `data.message` if it is false. I believe I do this in some of the other AJAX calls.